examples/MomentoRedisLambdaExample/MomentoRedisExampleLambdaHandler/src/MomentoRedisExampleLambdaHandler/Function.cs

```diff
@@ -0,0 +1,300 @@
+using Amazon.Lambda.Core;
+using StackExchange.Redis;
+using Momento.Sdk;
+using Momento.Sdk.Auth;
+using Momento.Sdk.Config;
+using Momento.Sdk.Responses;
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace MomentoRedisExampleLambdaHandler;
+
+public class Input
+{
+    public bool RunMomentoWithCompression { get; set; }
+    public bool RunMomentoCompressionWithSleep { get; set; } 
+    public bool RunMomentoWithoutCompression { get; set; }
+    public bool RunRedisWithCompression { get; set; }
+    public bool RunRedisWithoutCompression { get; set; }
+}
+
+public record struct GetResult(
+    CacheGetResponse Response,
+    long ResponseSize,
+    long DecompressedResponseSize,
+    long ElapsedMillis,
+    long ElapsedMillisWithDecompression
+);
+
+public record struct SetResult(CacheSetResponse Response, long ElapsedMillis);
+
+public record struct RedisGetResult(RedisValue Response, long ElapsedMillis, long ElapsedMillisWithDecompression);
+
+public record struct RedisSetResult(bool Response, long ElapsedMillis);
+
+public class Function
+{
+    const string CacheName = "MomentoLambda";
+    const string RedisClusterEndpoint = "momentoredisexamplecluster.exmof5.ng.0001.usw2.cache.amazonaws.com:6379";
```

Comment:
Should extract this to a parameter

Reply:
Will extract these as parameters to lambda function